application/templates/static_site/corrections.html

```diff
@@ -19,7 +19,7 @@ <h2 class="heading-small">
                                topic_slug=corrected_measure_version.measure.subtopic.topic.slug,
                                subtopic_slug=corrected_measure_version.measure.subtopic.slug,
                                measure_slug=corrected_measure_version.measure.slug,
-                               version=published_version_with_correction.version) }}">
+                               version='latest' if published_version_with_correction.measure.latest_published_version.version == published_version_with_correction.version else published_version_with_correction.version) }}">
```

Comment:
Would this be slightly simpler as
```
if published_version_with_correction.measure.latest_published_version == published_version_with_correction
```
?

Reply:
Yeah, I had that originally and it worked. But I dunno. It felt slightly wonky as it should be an `is` check that they're the same object, but jinja doesn't support that. 🤔